src/quickgui/plugin/qgsquickphotopanel.qml

```diff
@@ -139,87 +150,113 @@ Drawer {
+
+        MouseArea {
+          anchors.fill: parent
+          onClicked:photoPanel.cancelEvent()
         }
 
-        // Confirm button
-        Rectangle {
-          id: confirmButton
-          visible: camera.imageCapture.capturedImagePath != ""
-
-          property int borderWidth: 5 * QgsQuick.Utils.dp
-          width: buttonSize
-          height: buttonSize
-          color: "white"
-          border.color: photoPanel.borderColor
-          anchors.right: parent.right
-          anchors.verticalCenter: parent.verticalCenter
-          border.width: borderWidth
-          radius: width*0.5
-          antialiasing: true
-
-          MouseArea {
-            anchors.fill: parent
-            onClicked: {
-              captureItem.saveImage = true
-              photoPanel.visible = false
-              photoPanel.lastPhotoName = QgsQuick.Utils.getRelativePath(camera.imageCapture.capturedImagePath, photoPanel.prefixToRelativePath)
-              confirmButtonClicked(photoPanel.prefixToRelativePath, photoPanel.lastPhotoName)
-            }
-          }
+        Image {
+          fillMode: Image.PreserveAspectFit
+          anchors.centerIn: parent
+          sourceSize.height: imageButtonSize
+          sourceSize.width: imageButtonSize
+          height: imageButtonSize
+          source: photoPanel.cancelButtonIcon
+        }
+      }
+
+      // Confirm button
+      Rectangle {
+        id: confirmButton
+        visible: camera.imageCapture.capturedImagePath != ""
 
-          Image {
-            fillMode: Image.PreserveAspectFit
-            anchors.centerIn: parent
-            sourceSize.height: imageButtonSize
-            sourceSize.width: imageButtonSize
-            height: imageButtonSize
-            width: imageButtonSize
-            source: photoPanel.confirmButtonIcon
+        property int borderWidth: 5 * QgsQuick.Utils.dp
+        width: buttonSize
+        height: buttonSize
+        color: "white"
+        border.color: photoPanel.borderColor
+        anchors.right: parent.right
+        anchors.verticalCenter: parent.verticalCenter
+        border.width: borderWidth
+        radius: width*0.5
+        antialiasing: true
+
+        MouseArea {
+          anchors.fill: parent
+          onClicked: {
+            captureItem.saveImage = true
+            photoPanel.visible = false
+            photoPanel.lastPhotoName = QgsQuick.Utils.getRelativePath(camera.imageCapture.capturedImagePath, photoPanel.prefixToRelativePath)
+            confirmButtonClicked(photoPanel.prefixToRelativePath, photoPanel.lastPhotoName)
           }
         }
+
+        Image {
+          fillMode: Image.PreserveAspectFit
+          anchors.centerIn: parent
+          sourceSize.height: imageButtonSize
+          sourceSize.width: imageButtonSize
+          height: imageButtonSize
+          width: imageButtonSize
+          source: photoPanel.confirmButtonIcon
+        }
+      }
+    }
+
+    Rectangle {
+      id: backButton
```

Comment:
I would suggest to remove this circle - the back arrow is not properly aligned and it does not look great... better to have outline with the icon just like in the case of camera icon. And we will also get rid of the scaleRatio...